SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -391,7 +392,19 @@ void USpatialConnectionManager::FinishConnecting(Worker_ConnectionFuture* Connec
 				const USpatialGDKSettings* Settings = GetDefault<USpatialGDKSettings>();
 				SpatialConnectionManager->WorkerConnection = NewObject<USpatialWorkerConnection>();
 
-				SpatialConnectionManager->WorkerConnection->SetConnection(NewCAPIWorkerConnection, MoveTemp(EventTracing));
+				FComponentSetData SetData;
+
+				for (Worker_ComponentId i = 0; i < 100; ++i)
+				{
+					SetData.ComponentSets.FindOrAdd(SpatialConstants::WELL_KNOWN_COMPONENT_SET_ID).Add(i);
+				}
+				for (Worker_ComponentId i = 100; i < 100000; ++i)
```

Comment:
Could grab the next available ID from the schema database to get a better upper bound.

Reply:
if we can access schema database can't we just grab the component sets?

Reply:
I don't think all the ones that are hand written. Although we could do stuff from the schema bundle via the schema database. Although right now we don't have interesting component sets.